aip/general/0127.md

```diff
@@ -92,6 +92,17 @@ message CreateBookRequest {
   - The `body` **must not** be a `repeated` field.
   - Fields **should not** use the `json_name` annotation to alter the field
     name in JSON, unless doing so for backwards-compatibility reasons.
+  - In any given HTTP request, if a field is represented in both the body
+    and the URI, the values **must** be identical. The server behavior when
+    this constraint is violated is unspecified.
+- Eligible fields which are not represented in the URI path or the body are mapped
+  to URI query parameters.
+  - Only singular primitive, repeated primitive, and singular message type fields
```

Comment:
It would be good to link to the source of truth https://github.com/googleapis/googleapis/blob/dbfbfdb38a2f891384779a7ee31deb15adba6659/google/api/http.proto#L61 .... but is there a web-friendly, auto-up-to-date view of this somewhere we can use?

Reply:
I've added a link to the proto (in the master branch, which will be redirected accordingly if we rename it to main) earlier - I think that's probably clearer than trying to link to individual bits of documentation for each section.